lib/avro/builder/aliasable.rb

```diff
@@ -0,0 +1,20 @@
+require 'avro/builder/dsl_attributes'
+
+module Avro
+  module Builder
+
+    # This is a shared concern for objects that support aliases via the DSL.
+    module Aliasable
+      def self.included(base)
+        base.dsl_attribute :aliases do |*names|
+          if !names.empty?
```

Comment:
Do we need this conditional or can we just do `@aliases = names.flatten` in all cases?